src/lib/Libpython/pbs_python_svr_internal.c

```diff
@@ -2726,8 +2735,17 @@ pbs_python_populate_svrattrl_from_python_class(PyObject *py_instance,
 					continue;
 				}
 
-				strncpy(the_resc, resc, sizeof(the_resc)-1);
-				strncpy(the_val, val, sizeof(the_val)-1);
+				if ( val_buf_size < strlen(val)) {
```

Comment:
please remove the space after the parenthesis.. there are other such extra spaces below as well.